frontend/src/component/operator-wizard/element-component/field-factory/DropdownField.js

```diff
@@ -32,7 +32,9 @@ const mapDispatchToProps = dispatch => {
 
 const stepName = "Components",
   itemName = "Smart Router",
-  selectedEnvPrefix = "rhdm";
+  rhdmEnvPrefix = "rhdm",
```

Comment:
Literal constants should be better capitalized. I've also prefer these to be declared somewhere else as they can be used outside of the context of the Dropdown.
This affects the itemName (SmartRouter), rhdmEnv, rhpamEnv